src/OrbitGl/PrimitiveAssembler.h

```diff
@@ -98,6 +98,9 @@ class PrimitiveAssembler {
 
   void AddCircle(const Vec2& position, float radius, float z, Color color);
 
+  void AddVerticalArrow(float x, float y_start, float y_end, float z, uint32_t width,
+                        float head_length_ratio, float head_width_ratio, Color color);
```

Comment:
I'd include these 2 parameter in a simple Vec2 called `arrow_head_size`

Reply:
Also, why ratio? Is the head of the arrow be longer depending on the lenght of the arrow? I'd have the same size independent of the lenght.

In addition, I think that giving the absolute values could be more easy to reuse this method.